pkg/provider/aci.go

```diff
@@ -756,10 +763,13 @@ func (p *ACIProvider) Ping(ctx context.Context) error {
 	return nil
 }
 
+func (p *ACIProvider) NotifyNodeStatus(ctx context.Context, cb func(*v1.Node)) {
```

Comment:
When VK creates a node it sets the node status to pending.
The node provider is resposible for telling when the node is ready.
Since the implementation here is a no-op, the node is never marked ready.

I think when creating the provider you can return a nil node provider and VK should use the default implementation which should work fine (or at least like before).

Reply:
Tried that, but it didn't solve the metrics issue.